core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -2639,59 +2639,65 @@ class ReplicaManagerTest {
       time = time,
       scheduler = time.scheduler,
       logManager = logManager,
-      quotaManagers = QuotaFactory.instantiate(config, metrics, time, ""),
```

Comment:
Was there a reason we created a separated one here? I noticed that we change the config above if zkMigration is enabled, but not sure if that makes a difference.

Reply:
I agree we should not change any test semantics here. I updated the PR to close the new created `quotaManager` explicitly.